src/main/java/it/gov/pagopa/pu/classification/dto/ClassificationDTO.java

```diff
@@ -0,0 +1,47 @@
+package it.gov.pagopa.pu.classification.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import java.time.OffsetDateTime;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class ClassificationDTO {
```

Comment:
what's this? a DTO to configure query filter?
Let's rename into ExportClassificationsFilterDTO

Don't ask it as body of the API, the API should list each field as queryParam (see how it has been done in DebtPosition)

You will create this DTO inside the controller in order to propagate the filters until the Repository

Reply:
you have still to rename the class!